contracts/zero-ex/contracts/src/features/interfaces/IMultiplexFeature.sol

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+
+  Copyright 2021 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.6.5;
+pragma experimental ABIEncoderV2;
+
+import "@0x/contracts-erc20/contracts/src/v06/IERC20TokenV06.sol";
+
+
+interface IMultiplexFeature {
+
+    // Parameters for `batchFill`.
+    struct BatchFillData {
+        // The token being sold.
+        IERC20TokenV06 inputToken;
+        // The token being bought.
+        IERC20TokenV06 outputToken;
+        // The amount of `inputToken` to sell.
+        uint256 sellAmount;
+        // The nested calls to perform.
+        WrappedBatchCall[] calls;
+    }
+
+    // Represents a call nested within a `batchFill`.
+    struct WrappedBatchCall {
+        // The selector of the function to call.
+        bytes4 selector;
+        // Amount of `inputToken` to sell.
+        // Setting the high-bit indicates that `sellAmount & LOW_BITS`
+        // should be treated as a `1e18` fraction of the current balance
+        // of `sellToken`, where `1e18+ == 100%` and `0.5e18 == 50%`, etc.
+        uint256 sellAmount;
+        // ABI-encoded parameters needed to perform the call.
+        bytes data;
+    }
+
+    // Parameters for `multiHopFill`.
+    struct MultiHopFillData {
+        // The sell path.
+        address[] tokens;
```

Comment:
Nit - could we elaborate a bit on this struct, ex:`[input_token, hop_token_1, ..., hop_token_N, output_token]`